gradle/corset.gradle

```diff
@@ -97,3 +99,36 @@ tasks.register('buildReferenceTestsZkevmBin', Exec) {
         return standardOutput.toString()
     }
 }
+
+// Task which generates the Trace.java file using go-corset.
```

Comment:
@letypequividelespoubelles This is the task which replaces all of the tasks that were in `trace-files.gradle` and also the `TraceFilesTask.groovy` task.